### PR TITLE
Add tests for titles with colon

### PR DESCRIPTION
**Pull request by icyflame**
The last comment on #1 explains the problem.

@sindresorhus Please review.

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -20,4 +20,14 @@ it('should get article title from a HTML document', function () {
 	assertHTML('tylerhenkel', 'Angular Fullstack 1.2.0 available now');
 	assertHTML('numediaweb', 'Web App with dream team; AngularJS, Cordova, Yeoman & Topcoat');
 	assertHTML('github-repo', 'urls-md');
+	assertHTML('youtube-wiki', 'Wikipedia: #Edit2014');
+});
+
+it('should get title containing colon from Youtube', function(){
```
Comment (sindresorhus) [inline test.js]: I don't see the point in this. It's just duplicating the above one. Also not interested in requiring network to run the tests.

**Comment by sindresorhus** [comment]
Happy to add the failing test, but remove the other one the does curl.

**Comment by icyflame** [comment]
@sindresorhus 

Failing test case added. I have removed the other one with curl.

File: test.js [hunk 2]
```diff
@@ -20,4 +20,6 @@ it('should get article title from a HTML document', function () {
 	assertHTML('tylerhenkel', 'Angular Fullstack 1.2.0 available now');
 	assertHTML('numediaweb', 'Web App with dream team; AngularJS, Cordova, Yeoman & Topcoat');
 	assertHTML('github-repo', 'urls-md');
+	assertHTML('youtube-wiki', 'Wikipedia: #Edit2014');
 });
+
```
Comment (arthurvr) [inline test.js]: Please remove this empty line :)

**Comment by icyflame** [comment]
@arthurvr Done! :)

**Comment by kevva** [comment]
Thanks yo!

**Pull request closed without merging**